### PR TITLE
[css-typed-om] Add a typed OM spec for ColorValue

### PR DESCRIPTION
**Pull request by wilddamon**
@shans This is a tidied-up version of @njlawton's draft spec for ColorValue. PTAL

### CODE REVIEW

File: css-typed-om/Overview.bs [hunk 1]
```diff
@@ -351,6 +351,29 @@ Issue: There are now 3 ways of creating Length values -
...
+---------------------------------------------
+<pre class='idl'>
+
+[Constructor(double r, double g, double b),
+ Constructor(double r, double g, double b, double a)]
+interface ColorValue : StyleValue{
+  readonly attribute double r;
+  readonly attribute double g;
+  readonly attribute double b;
+  readonly attribute double? a;
```
Comment (upsuper) [inline css-typed-om/Overview.bs]: Should `a` be nullable? I suppose it should just be `1.0` when not specified.

File: css-typed-om/Overview.bs [hunk 2]
```diff
@@ -351,6 +351,29 @@ Issue: There are now 3 ways of creating Length values -
 
 Issue: What happens if the provided DOMString or LengthValue for (e.g.) a SimpleLength is invalid?
 
+{{ColorValue}} objects {#colorvalue-objects}
+---------------------------------------------
+<pre class='idl'>
+
+[Constructor(double r, double g, double b),
+ Constructor(double r, double g, double b, double a)]
```
Comment (upsuper) [inline css-typed-om/Overview.bs]: I wonder whether we should allow constructing `ColorValue` with supported color keywords. Or is there any existing function to do so?

Comment (wilddamon) [inline css-typed-om/Overview.bs]: @shans would have a better answer to that?

**Comment by tabatkins** [comment]
Closing because we're instead porting over my previous attempt at adding a Color JS API from the Colors 4 spec.

**Pull request closed without merging**